shared/src/packages/users/libs/types/user-sign-up-response-dto.type.ts

```diff
@@ -1,6 +1,11 @@
-type UserSignUpResponseDto = {
+type UserAuthResponseDto = {
```

Comment:
Move it to a separate file, we will use it for sign-in and get authenticated-user too

Reply:
Will do!